### DISCUSSION TITLE
OIDC login issues after upgrade to

### DISCUSSION THREAD

**Discussion by AndrewMcLachlan** [datalust/seq-tickets#1168] [Help]
I have upgraded Seq from 2021.1.5307 to 2021.1.5425.

We are logging in using Azure Active Directory.

While existing sessions are working, new logins are failing. I pulled this from the Seq logs:

`{"@t":"2021-03-15T07:05:11.1547163Z","@mt":"Exception occurred while processing message.","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request.\r\n ---> System.Net.Http.WinHttpException (80072EE2, 12002): Error 12002 calling WINHTTP_CALLBACK_STATUS_REQUEST_ERROR, 'The operation timed out'.\r\n   at System.Threading.Tasks.RendezvousAwaitable``1.GetResult()\r\n   at System.Net.Http.WinHttpHandler.StartRequestAsync(WinHttpRequestState state)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.Http.HttpClient.SendAsyncCore(HttpRequestMessage request, HttpCompletionOption completionOption, Boolean async, Boolean emitTelemetryStartStop, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Authentication.OpenIdConnect.OpenIdConnectHandler.GetUserInformationAsync(OpenIdConnectMessage message, JwtSecurityToken jwt, ClaimsPrincipal principal, AuthenticationProperties properties)\r\n   at Microsoft.AspNetCore.Authentication.OpenIdConnect.OpenIdConnectHandler.HandleRemoteAuthenticateAsync()","EventId":{"Id":17,"Name":"ExceptionProcessingMessage"},"SourceContext":"Microsoft.AspNetCore.Authentication.OpenIdConnect.OpenIdConnectHandler","RequestId":"800191cf-0000-f500-b63f-84710c7967bb","RequestPath":"/aad"}
{"@t":"2021-03-15T07:05:11.1556096Z","@mt":"OpenID Connect authentication failed with {Error} / {ErrorDescription}","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request.\r\n ---> System.Net.Http.WinHttpException (80072EE2, 12002): Error 12002 calling WINHTTP_CALLBACK_STATUS_REQUEST_ERROR, 'The operation timed out'.\r\n   at System.Threading.Tasks.RendezvousAwaitable``1.GetResult()\r\n   at System.Net.Http.WinHttpHandler.StartRequestAsync(WinHttpRequestState state)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.Http.HttpClient.SendAsyncCore(HttpRequestMessage request, HttpCompletionOption completionOption, Boolean async, Boolean emitTelemetryStartStop, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Authentication.OpenIdConnect.OpenIdConnectHandler.GetUserInformationAsync(OpenIdConnectMessage message, JwtSecurityToken jwt, ClaimsPrincipal principal, AuthenticationProperties properties)\r\n   at Microsoft.AspNetCore.Authentication.OpenIdConnect.OpenIdConnectHandler.HandleRemoteAuthenticateAsync()","Error":"No error provided","ErrorDescription":"No extended description available","RequestId":"800191cf-0000-f500-b63f-84710c7967bb","RequestPath":"/aad"}`

While I suspect it might be to do with our corporate proxy, I was wondering has changed between versions that might have caused this issue?

**Comment by nblumhardt** [comment]
Hi Andrew,

This looks like it will be due to https://github.com/datalust/seq-tickets/issues/1145 - the newer build will use Windows proxy settings, while the older build (because of a regression we introduced in 2020.5) did not.

Are you expecting Seq to use the proxy, and is there a chance it's stale proxy configuration on the server machine? We may need to look at adding a setting to turn use of the system proxy on/off, otherwise.
